main.py

```diff
@@ -121,31 +121,33 @@ def registration():
         with open("access_token.txt") as f:
             if access_token != f.read():
                 print("bad token")
```

Comment:
For production environments, I'd prefer to see [`logging`](https://docs.python.org/3/library/logging.html) used. This could simply be made an `INFO`, or even `DEBUG` log entry.